src/AbstractSerializer.php

```diff
@@ -38,7 +38,7 @@ protected static function getLine(StreamInterface $stream)
     {
         $line    = '';
         $crFound = false;
-        while (! $stream->eof()) {
+        while (!$stream->eof()) {
```

Comment:
Honestly, this is a personal preference thing, and I've actually been changing these in the ZF2 code base to `! $...` when I come across them. Let's leave these as-is, please.